examples/monitor-with-externalConfigMap/README.md

```diff
@@ -0,0 +1,36 @@
+# A example describe monitoring with external configMap
+
+> **Note:**
+>
+> This setup is for teach user how to monitor with an external configMap
```

Comment:
> This document is to show how to mount external configmap for prometheus in `TidbMonitor`. This would help to customize user's own prometheus configurations or rules.

Reply:
done